partitura/io/importmusicxml.py

```diff
@@ -841,6 +841,9 @@ def _handle_direction(e, position, part, ongoing):
             # first child of direction-type is dynamics, there may be subsequent
             # dynamics items, so we loop:
             for child in direction_type:
+                # check if child has no children, in which case continue
```

Comment:
could you add a small test on when this was happening? Ideally a score with only the problematic measure inside. So we are sure that future partitura changes will not undo the fix

Reply:
Also, a probably nicer solution, next() accepts a default value. 
So if you do next(iter(), None) this will return None when the iterator is empty, and the None will be correctly handled by the next if statement

Reply:
https://stackoverflow.com/questions/14250184/default-value-for-next-element-in-python-iterator-if-iterator-is-empty/14250244#14250244

Reply:
There is an issue with writing a test for this case since the musicxml export will not be the same as the import. This causes the test function  `test_import_export` to fail.

Reply:
Very annoying. But then just don't add the piece to MUSICXML_IMPORT_EXPORT_TESTFILES. Instead create a new collection like MUSICXML_IMPORT_TESTFILES

Reply:
I added it implicitly to something else, so it can go through the coverage function. I don't think it should have its own file or function, it is too much.